cluster/meta/meta.pkg.crossplane.io_functions.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     singular: function
   scope: Namespaced
   versions:
-  - name: v1beta1
+  - name: v1
```

Comment:
This also seems unrelated to the rest of the PR. Please do it in a separate commit.

Reply:
No, I think I messed something up with the generator. I removed it.